api/v1alpha1/scaledobject_types.go

```diff
@@ -54,7 +54,6 @@ type AdvancedConfig struct {
 
 // HorizontalPodAutoscalerConfig specifies horizontal scale config
 type HorizontalPodAutoscalerConfig struct {
-	ResourceMetrics []*autoscalingv2beta2.ResourceMetricSource `json:"resourceMetrics,omitempty"`
 	// +optional
 	Behavior *autoscalingv2beta2.HorizontalPodAutoscalerBehavior `json:"behavior,omitempty"`
```

Comment:
If `Behavior` isn't propagated to resource scaler for HPA, how could it make `scaleUp` and `scaleDown` take effect? Currently sacleUp behavior doesn't work as expected for resource (cpu/memory) metrics https://github.com/kedacore/keda/issues/1244.

Reply:
discuss in #1244.

please upgrade to keda-2.0.0-rc and try again.